fe/topology.py

```diff
@@ -21,6 +21,12 @@ def standard_qlj_typer(mol):
     in a relatively simple and forcefield independent way. The
     parameters here roughly follow the Smirnoff 1.1.0 Lennard Jones
     types.
+
+    These values are taken from ff/params/smirnoff_1_1_0_cc.py, rounding down
+    to two decimal places for sigma and one decimal place for epsilon.
```

Comment:
Nice! Confirmed this is an accurate description of how the previous parameters were obtained? https://github.com/proteneer/timemachine/pull/450#discussion_r653976062